.github/workflows/build-and-test.yaml

```diff
@@ -7,8 +7,16 @@ on:
   workflow_dispatch:
 
 jobs:
+  prevent-running-on-forks:
+    name: Don't run scheduled jobs on forks
+    runs-on: ubuntu-latest
+    if: ${{ (github.event_name != 'schedule') || (github.event_name == 'schedule' && github.repository == 'autowarefoundation/autoware.universe') }}
+    steps:
+      - run: "true"
+
   build-and-test:
     if: ${{ github.event_name != 'push' || github.ref_name == github.event.repository.default_branch }}
+    needs: prevent-running-on-forks
```

Comment:
I guess some fork users want to run scheduled build workflows, so we should remove this.
Otherwise, every user needs to fix the workflow file, it's annoying.

Reply:
Okay, removed: e032907cf